swift/apple/FirezoneNetworkExtension/Adapter.swift

```diff
@@ -72,7 +47,19 @@ class Adapter {
   /// Network routes monitor.
   private var networkMonitor: NWPathMonitor?
 
-  /// Private queue used to synchronize access to `WireGuardAdapter` members.
+  /// Used to avoid path update callback cycles on iOS
+  #if os(iOS)
+    private var gateways: [Network.NWEndpoint] = []
+  #endif
+
+  /// Track our last fetched DNS resolvers to know whether to tell connlib they've updated
+  private var lastFetchedResolvers: [String] = []
+
+  /// Used to avoid needlessly sending reconnects to connlib
+  private var primaryInterfaceName: String?
+
+  /// Private queue used to ensure consistent ordering among path update and connlib callbacks
+  /// This is the primary async primitive used in this class.
   private let workQueue = DispatchQueue(label: "FirezoneAdapterWorkQueue")
```

Comment:
These are scoped to the application or the process, right? I do see the Apple docs saying e.g. `com.example.myqueue` but hopefully no other app sees these queues.